test/examples/Makefile.am

```diff
@@ -22,3 +22,27 @@ installcheck-local:
 	$(CC) -o ucp_client_server $(examplesdir)/ucp_client_server.c -lucp $(EXAMPLE_CC_FLAGS) -pedantic
 	$(CC) -o ucx_profiling     $(examplesdir)/ucx_profiling.c     -lucs $(EXAMPLE_CC_FLAGS) -Wall -lm
 	$(RM) *.o uct_hello_world ucp_hello_world ucp_client_server ucx_profiling
+
+if HAVE_EXAMPLES
+
+EXAMPLE_INCLUDE_FLAGS = -I$(abs_top_srcdir)/src -I$(abs_top_builddir)/src
+
+noinst_PROGRAMS = ucp_hello_world uct_hello_world ucx_profiling ucp_client_server
+
+ucp_hello_world_SOURCES   = ucp_hello_world.c
+ucp_hello_world_CFLAGS    = $(BASE_CFLAGS) $(EXAMPLE_INCLUDE_FLAGS)
```

Comment:
maybe set global AM_CPPFLAGS , AM_CFLAGS?

Reply:
actually BASE_CFLAGS is such global var, and every component adds own portion of flags specific for component